modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -519,6 +519,15 @@ definition {
 			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
 	}
 
+	@summary = "View activity status on the MDF request detail page"
+	macro viewActivityStatus {
+		AssertTextEquals(
+			key_statustitle = ${statustitle},
+			key_statusvalue = ${statusvalue},
+			locator1 = "PRMMDFRequest#ACTIVITY_STATUS",
+			value1 = ${activityStatus});
+	}
```

Comment:
Hi @marinaasant0s, use `AssertElementPresent` here. Since the HTML isn't very good and have some blank spaces after and before the texts, the `AssertTextEquals` can fail the tests in some cases

![image](https://user-images.githubusercontent.com/69267630/232891614-31591f34-f381-4684-881c-ca67d9500e73.png)

Reply:
Also, there are some typos in variable definitions `key_statustitle` > `key_statusTitle`. `statustitle` > `statusTitle` and the same to `statusvalue`. Remember to use camelCase.